lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './dish.rb'
+require_relative './order.rb'
+
+class Menu
+  attr_reader :menu_list
```

Comment:
I'm a bit confused as to what menu_list is. I can't see another instance variable with the same name